core/nuxeo-connection.js

```diff
@@ -40,7 +209,7 @@ import './nuxeo-element.js';
    *
    * @memberof Nuxeo
    */
-  class Connection extends Nuxeo.Element {
+  class ConnectionElement extends ConnectionMixin(Nuxeo.Element) {
```

Comment:
So `Connection` used to be an element class that did everything and is now a vanilla JS class that only handles the "core" part. Can't that be a problem? It won't if everyone is just using these as custom elements, but if someone is doing something different with these classes there might be a discrepancy.
Not sure if we're concerned about that but a safer solution would be to keep the same class name on the element and use a different one for the "core" class part.
(same with `Operation`)